docs/how-to-create-a-plugin.md

```diff
@@ -32,3 +32,14 @@ Learn more about the [Transformer API](/docs/transformer-api/)
 ## Plugins for other plugins
 
 Some plugins, like the `@gridsome/transformer-remark`, can have its own plugins. They can be named `gridsome-remark-*` or `@username/gridsome-remark-*`.
+
+## Testing your plugin
+
+1. [Create a new Gridsome project](/docs/#2-create-a-gridsome-project) (if you don't want to test with an existing one)
+2. Open a terminal in your plugin project directory
+3. `npm link my-plugin-name` or `yarn link my-plugin-name`
```

Comment:
`yarn link my-plugin-name` in plugin project dir gives error `No registered package found called "my-plugin-name".`

Maybe just `yarn link` here?